src/translate.py

```diff
@@ -2437,6 +2459,7 @@ def process_instr(instr: Instruction) -> None:
             process_instr(instr)
 
     if branch_condition is not None:
+        assert not isinstance(branch_condition, CommaConditionExpr)
```

Comment:
this assert isn't really needed

Reply:
You're probably right, let me see what happens when I get rid of it